code/game/gamemodes/cult/items/tome.dm

```diff
@@ -90,9 +90,15 @@
 		if(use_check_and_message(scribe))
 			return
 
-		scribe.visible_message(SPAN_CULT("[scribe] slices open their palm with a ceremonial knife, drawing arcane symbols with their blood..."))
-		playsound(scribe, 'sound/weapons/bladeslice.ogg', 50, FALSE)
-		scribe.drip(4)
+		scribe.visible_message(SPAN_WARNING("[scribe] closes their eyes and presses their hand against a page in their book, then takes out a piece of chalk from the book and draws strange symbols on the ground..."), SPAN_CULT("You close your eyes and worshipfully pierce your hand against the spikes on the rune's page, imbuing the tome's chalk with its magic and your blood.  You feel His presence wash over you, sealing the wounds as you begin drawing with the chalk."))
+		
+		//piercing your hand on the spikes should add your blood to it.  examine proc keeps it from being obviously blood-stained.  don't know how to handle it being bloodied by normal means... I guess tomes magically suck in all the blood that gets on them? : P -MalMalmulam
+		if(!blood_DNA)
+			blood_DNA = list()
+		if(!blood_DNA[scribe.dna.unique_enzymes])
+			blood_DNA[scribe.dna.unique_enzymes] = scribe.dna.b_type
```

Comment:
I don't understand the point of making the runes not look blood infused if you're making them bloody anyway.

Reply:
I realized I never answered this here, so basically, it's "secret" blood magic - still uses blood, but at least people won't be going OH NO IT'S BLOOD THEY'RE CRAZY.